README.md

```diff
@@ -47,8 +48,29 @@ avatar = "https://yoursite.com/avatar.png"
 url = "https://github.com/yourusername"
 label = "GitHub"
 icon = "fa-brands fa-github"
+
+# Optional: Add extras section for presentations, talks, resources, etc.
+[[extras]]
+label = "PyCon 2024 - My Talk"
+url = "https://example.com/slides"
+icon = "fa-solid fa-presentation"
+date = "May 15, 2024"  # Optional date field
```

Comment:
Make date required, and sort entries by most recent first.

If there are more than 5, place the rest behind an expandable details section.